go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/containers/podman/v4
 
-go 1.20
+go 1.21
+
+toolchain go1.21.6
```

Comment:
This is going to break on all envs that don't have go1.21.6 . Fedora 38 has 1.20.13 . I guess I could disable fedora 38 in packit and on copr. We won't be shipping podman5 on F38 anyway. Even centos 8 might be doable. Need to check with the centos folks on centos 9 though.

Reply:
and debian-13 is also one patch release behind.

Reply:
Lets hold off on this.